### ISSUE TITLE
Split of base and novel categories for open-vocabulary test

### ISSUE THREAD

**Issue by seanzhuh** [OpenMask3D/openmask3d#26]
Hi, in Tab. 3, 53 classes that are semantically close to ScanNet20 are grouped to base categories, and the other 147 classes are regarded as novel ones. Since neither the paper or the code mentions how this grouping is done, could you please share the detailed names for base/novel classes ? And the pretrained mask module weights trained on ScanNet20 (there is only a checkpoint trained on ScanNet200 train split)?

**Comment by seanzhuh** [comment]
Does the weights trained on ScanNet200 predicts class-specifc logits that are later simply discarded or Mask3D is retrained with a binary classification branch? The predicated binary masks with or without classification branch might be different.

**Comment by aycatakmaz** [comment]
Hi,
Thank you for your interest in our work! We have included the base class list in this issue: https://github.com/OpenMask3D/openmask3d/issues/2#issuecomment-1807150787 - all remaining classes were considered as novel categories.

For experiments in Tab. 3 where we present results with the mask module trained on ScanNet20, we used the [this checkpoint](https://omnomnom.vision.rwth-aachen.de/data/mask3d/checkpoints/scannet/scannet_val.ckpt) from Mask3D, [please refer to this page](https://github.com/JonasSchult/Mask3D?tab=readme-ov-file#scannet-v2:~:text=visualizations-,ScanNet%20200,-Dataset) for details.

In all of our experiments, during training we use checkpoints pre-trained in a supervised manner where the model predicts class-specific logits that are simply discarded. We basically discard everything related to the logits and only keep the binary masks. 

This is indeed correct:
> The predicated binary masks with or without classification branch might be different.

We have a follow up work [Segment3D](https://segment3d.github.io/) where the model is trained in a fully class-agnostic setting without class assignments.

Hope this brings some clarity!
Ayca

**Comment by seanzhuh** [comment]
Thanks for your effort put into maintaining the issues of this repo after 2 years!
